game/views.py

```
@@ -1475,6 +1476,29 @@ def analyze_game_view(request):
+    "Forks",
+    "Pins",
+    "Skewers",
+    "Discovered Attacks",
+    "Pawn Structures",
+    "King Safety",
+    "Piece Activity",
+    "Basic Endgames",
+)
+
+
+def _lesson_name_from_slug(lesson_slug):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_lesson_name_from_slug`

(ANN202)